### ISSUE TITLE
Add information about running benchmark to User Guide

### ISSUE THREAD

**Issue by jchodera** [openmm/openmm#3346]
I can't seem to [find anything about benchmarking in the User Guide](http://docs.openmm.org/latest/userguide/search.html?q=benchmark). Can we add something about what benchmarking tools we have available and how to run them?

**Comment by peastman** [comment]
Good idea.

**Comment by lukeczapla** [comment]
I'm gonna agree and be a bit more to the point - what has been done much at all since 2014?  I would love an 'examples' folder to go with the developer's guide so I'm not just reading these great rst files in VIM and looking at the little you provide like I'm some dude who'd even mess with Visual Studio, ugh, come on - needzzz some helpzzz man, yea I love your little irrXML example and that's way smarter than all that tedious work I did with my stupid Excalibur from 2012-2013 but we need some docs to up the game here, my friend. 

Even the basic examples folder is SOOOOOOO out of date, it's the same one from back then essentially so we can up this in every way possible and show them what's the way to do this like a boss.   I mean I had no trouble re-implementing PCA for Deeplearning4J and at least they had a whole Slack channel with people you could talk to for that, files labelled for dummies would help!!!

**Comment by lukeczapla** [comment]
and my Excalibur just pushed 23 ns/day anyway vs. 21 ns/day so it's just about integrating that guy with the all-atom coarse-grained models of DNA that are way more feasible than probably even AMBER parmbsc1 because they keep the conformations reasonable.

**Comment by peastman** [comment]
What rst files are you reading in vim?  Have you looked at the [online documentation](https://openmm.org/documentation)?

**Comment by lukeczapla** [comment]
yes I did, obviously :) no brainer there!  you must be getting old if you don't even remember openmm/docs-source/developersguide in your repo, it's pretty good!

**Comment by lukeczapla** [comment]
you know I'm just playing - but not about the requests here, they would be super helpful and then someone else can actually jump in and it's not just a 'dead' project barely plodding along with nothing new to say.

**Comment by lukeczapla** [comment]
we didn't even use github repos much back in 2012 but it's a new game and we don't just sit still here because maybe funding went to something else - it's still important and that's BS marketing like the CUDA vs. OpenCL thing.  eff them if they don't have any loot to spare anymore.

**Comment by lukeczapla** [comment]
I know all my junk on simtk was like superdead, so this is it here right?

**Comment by lukeczapla** [comment]
we like to read rst files in vim but if you don't provide the barebones example how does one even learn anything? seems like locking out everyone else didn't do ya that great here, this is the open source channel.

**Comment by lukeczapla** [comment]
the linux kernel will take an update from a 16 yr old, I don't see why the docs had to be so convoluted as if people weren't actually really trying to do this and help, but nobody really ever helps if it's not about a $$

**Comment by lukeczapla** [comment]
I'm sure I prolly hit up Pande or one of those doods when I was on unemployment for a very brief time ages ago and they definitely had no interest in actually helping other people contribute to the project even beyond the money - unemployment was actually pretty good $ and fun, 5x the output.. prolly could've added a feature for free or something, that was the deal with DL4j, summers off from teaching.

**Comment by lukeczapla** [comment]
like trust us man, what you do is not THAT complicated here, it's just stuff other people already invented and we would like to help out here if you give us a chance here.. I had a blast with this little ASM interrupt table little book when I was 12, effing real mode, imagine that? ;)

**Comment by lukeczapla** [comment]
you got a pattern, a simple example folder with just CPU & OpenCL examples that do very little would work WONDERS here!  stupid NVIDIA compiler is just a pain in the ass but a guy over here is gonna experiment with this A100 thing and benchmarks are totally fun! John got one, I got two already (C++ and Python), etc... we be having a blast enough but hidden information is just like totally not part of the open source ethos... says anonymous person, I hate being a real name.

**Comment by j-wags** [comment]
I think some of the posts on this issue may be running contrary to the
repo’s code of conduct, specifically the one about the age of the
maintainer, but generally the lack of professional tone. This is a useful
topic to discuss, but the relevant information is spread over several
posts, making it hard to discuss specific points or come up with a plan of
action.

**Comment by lukeczapla** [comment]
I'm totally up for this too though, so I hope that we have a location to really get into this because I need some help over here and I just like to get a couple lol's in too, it's called the internet right?  please let me know if I've overspammed but I think there's a couple good points in here to unpack so please write down the plan of action because I have the specific request in the tone of the free speech principles that really should be guiding this, but definitely throw me some rules if you need to be the most formal - that's why other projects have Slack channels and such.......

**Comment by lukeczapla** [comment]
I do not want to be discriminated against for my particular form of brain here.   So lay out your law Mister.   Thank you!

**Comment by raimis** [comment]
Related issue: https://github.com/openmm/openmm-org/issues/84

**Comment by peastman** [comment]
Let's please keep the discussion focused on the task at hand.  This issue tracker is a tool we rely on to do our jobs, not a forum for free discussion.  This issue is about adding a section to the user guide describing how to run benchmarks.  Anything that isn't directly related to that belongs somewhere else.  If you want to open a separate issue to propose specific changes to the examples folder, that would be great.

**Comment by lukeczapla** [comment]
point taken. go discuss this too though, totally legit also and the copyrights on the makefiles I fixed up for u were 2009 so 2012-2013 was my best guess on when I just dropped on simtk a while back and fell asleep - weren't you removing those stupid simtk imports and isn't it cool that I did that in my codez already for you guys so that it's modern and up to date - python is decent, C++ died and CUDA/OpenCL must seriously be plodding along with just 1 dev
